### PR TITLE
build: take into account PKG_VSN env variable in rebar.config.erl

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: rebar.config.erl [hunk 1]
```diff
@@ -482,10 +482,15 @@ emqx_etc_overlay_per_edition(ee) ->
...
 get_vsn(Profile) ->
-    %% to make it compatible to Linux and Windows,
-    %% we must use bash to execute the bash file
-    %% because "./" will not be recognized as an internal or external command
-    os_cmd("pkg-vsn.sh " ++ atom_to_list(Profile)).
+    case os:getenv("PKG_VSN") of
+        false ->
+            %% to make it compatible to Linux and Windows,
+            %% we must use bash to execute the bash file
+            %% because "./" will not be recognized as an internal or external command
```
Comment (zmstone) [inline rebar.config.erl:489]: this comment should be moved down as the function doc for `os_cmd/1`
ref: 65bb71ccca

Comment (zmstone) [reply rebar.config.erl:489]: later changed in this commit:
4e64010a3b
but left the comment behind.

**Comment by zmstone** [comment]
can we get more context for why this change ?

**Comment by id** [comment]
> can we get more context for why this change ?

right now you can't build emqx from source code tar.gz artifact because it tries to resolve version via git

**Review by thalesmg — APPROVED** [review]

**Pull request merged**